include/exiv2/basicio.hpp

```diff
@@ -79,7 +79,7 @@ class EXIV2API BasicIo {
     @return Number of bytes written to IO source successfully;<BR>
         0 if failure;
    */
-  virtual size_t write(const byte *data, size_t wcount) = 0;
```

Comment:
O_O ... is not this controlled by clang-format? I am pretty sure the is something in clang-format to control the indentation of `&` and `*` in function parameters.

Was this something that you changed manually?

Reply:
clang-format did this, yes. In the current setup, clang-format decides the order based on the most used order. Probably should disable DerivePointerAlignment.

Reply:
I agree, probably we should use something like:

```
# Force pointers to the type for C++.
DerivePointerAlignment: false
PointerAlignment: Left
```
To enforce the same style over the whole project. @hassec what's your opinion on this point? 

For me it is fine to accept this PR as it is, and then create another PR to address those clang-format changes.

Reply:
Fully agree with you  @piponazo :+1:       
And yes, I also think we can move this forward independently and fix `clang-format` in a different PR

Reply:
Pointer alignment should be left, yes.